sympy/integrals/integrals.py

```diff
@@ -173,8 +176,8 @@ def transform(self, x, u):
         where `F(x)` is the inverse of `f(x)` and the limits and integrand have
         been corrected so as to retain the same value after integration.
 
-        Notes
-        =====
+        Explanation
+        ===========
 
         The mappings, F(x) or f(u), must lead to a unique integral. Linear
         or rational linear expression, `2*x`, `1/x` and `sqrt(x)`, will
```

Comment:
These formulas use single backticks, but they are written like code. They should either be rewritten with standard LaTeX, or fixed to use double backticks.

Reply:
Done.